pkg/pipelineascode/pipelinesascode_github_test.go

```diff
@@ -473,17 +475,6 @@ func TestRun(t *testing.T) {
 				logmsg := log.FilterMessageSnippet(tt.expectedLogSnippet).TakeAll()
 				assert.Assert(t, len(logmsg) > 0, "log messages", logmsg, tt.expectedLogSnippet)
 			}
-
-			if tt.finalStatus != "skipped" {
```

Comment:
can we test that the PR has been created then if we can't test for repo creation anymore?